lib/command.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
Ah, if it threw an error, then you're probably right. I just really like creating closures in js to avoid polluting the global namespace (ie when browserify concatenates files for example)